app/controllers/DiagnosticReportController.scala

```diff
@@ -41,4 +41,13 @@ trait DiagnosticReportController extends BaseController {
       case e => NotFound
     }
   }
+
+  def getAllApplications() = Action.async { implicit request =>
```

Comment:
Interesting point. It is inconsistent in some ways to put a return type. I notice that none of the controllers in fset-faststream have return types. In many ways, there is a reduced value as most of the return types have to be Action[AnyContent]